src/index.css

```diff
@@ -145,74 +195,62 @@ a:hover {
   box-shadow: 0 5px 5px 0 var(--shadow), 0 4px 0px 0 var(--main-shadow);
 }
 
-.nav-link .fa-github {
-  width: 2.1em;
-  height: 2.1em;
-}
-
-.navbar-light .navbar-nav .nav-link:hover {
-  border-bottom: solid var(--secondary-accent) .17em;
-}
+/* ############################# */
+/* #PAGE                       # */
+/* ############################# */
 
-.navbar-light .navbar-nav .nav-item:last-child .nav-link {
-  margin-right: 0;
-}
-
-.navbar-brand {
-  content: ' ';
-  background-image: url("/img/DraCor-white.svg");
-  background-repeat: no-repeat;
-  width: 4em;
-  height: 4em;
-  margin: 1em 0 1em 2em;
-}
-
-.nav-link span {
-  display: none;
-}
-
-.navbar-collapse .ml-auto, .navbar-collapse .mx-auto {
-    margin-right: 2em;
-    margin-left: 0 !important;
+.content {
+  position: relative;
 }
 
-.navbar-collapse .git-icon {
-  margin: 1em 2em 1em 0;
+.content::before {
```

Comment:
What does this do?

Reply:
This is a fallback cover for when `Headroom` fails to (re)calculate the correct height after viewport resizing and leaves a visible gap between the navbar and page heading:

<img width="1384" alt="Bildschirmfoto 2020-11-23 um 12 52 58" src="https://user-images.githubusercontent.com/53089225/99959620-9fd25d80-2d8b-11eb-8e0e-e3b26429d24f.png">

Although it occurs rarely, I would keep this fallback until all `Headroom` issues are resolved. Could be linked to issue  #109 btw.

Reply:
Ok, thanks I left a comment in the code to document this: e8eb0f67a43808597efee7b22294d279e8f5619a.